test/bddtests/fixtures/config/couchDB/config.ini

```diff
@@ -0,0 +1,8 @@
+#
+# Copyright IBM Corp, SecureKey Technologies Inc. All Rights Reserved.
```

Comment:
There were really two authors of a two line file?

Reply:
done